quickstarts/lambdatest/config.yml

```diff
@@ -0,0 +1,61 @@
+id: f5d5d638-24ee-4a1c-859f-9cbd4d8cabbc
+slug: lambdatest
+title: LambdaTest
+description: |
+  ## About LambdaTest
+​
+  LambdaTest is an AI-powered test orchestration and execution platform to run manual and automated tests at scale. With LambdaTest, developers and testers can test their websites and mobile applications across 3000+ real browsers, devices, and operating systems.
+​
+  ## About LambdaTest and New Relic Integration
+​
+  The integration of LambdaTest with New Relic offers a seamless experience for users aiming to optimize and monitor their stats during automated test execution. By bridging the gap between testing and performance monitoring, this integration brings forward an out-of-the-box dashboard that enables users to visualize and understand their testing metrics in real-time, set against the backdrop of broader application performance insights offered by New Relic.
+​
+  ## LambdaTest Dashboard and Visualization
+​
+  Upon successful integration, the LambdaTest Dashboard in New Relic becomes a hub of insights. This tailored interface showcases key testing metrics, offering a comprehensive perspective on your automated testing endeavors. Gain a high-level view of test executions, failures, and compatibility issues, ensuring you're always in the know about the quality of your web or mobile applications.
+​
+  ### Test Overview
+​
+  - Holistic view of automated testing activities.
+  - Displays key metrics such as test execution counts, pass rates, durations, etc.
+  - Insights on website compatibility across different browsers and devices.
+  - Immediate understanding of application's cross-browser functionality.
+​
+​
+  ### Test Errors Overview
+​
+  - Focuses on discrepancies and anomalies in tests.
+  - Deep insights into test failures.
+  - Highlights the nature and frequency of errors.
+  - Provides information on environments where errors occurred.
+​
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+​
+summary: |
+  The LambdaTest Quickstart offers a streamlined entry into automated testing. With just a few clicks, users can instantly set up, monitor, and evaluate their application's performance across multiple browsers and devices, ensuring seamless user experiences everywhere.
+level: Community
+authors:
+  - Prateek Saini
+  - LambdaTest
+keywords:
+  - lambdatest
+  - logs
+  - newrelic partner
+  - devops
+  - cicd
+  - testing
+  - cross-browser
+  - featured
+​
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+​
+installPlans:
+  - third-party-lambdatest
+dataSourceIds:
+  - lambdatest
```

Comment:
This data source does not appear to exist, could you add it to this pull request?

Reply:
data source added